### PR TITLE
Add blog post Build your serverless functions with OpenFaas on Azure

### PR DESCRIPTION
**Pull request by sajeetharan**
Signed-off-by: Sajeetharan Sinnathurai sajeefx@gmail.com

## Description
Added a link to blogpost about on "Building microservices on azure platform with OpenFaas"

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required]
- [x] Requested

## How Has This Been Tested?
N.A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Blog

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ It would be great to hear from you especially if you have any of the above and w
 
 | Blog/repo name and description                                          | Author       | Site     | Date        |
 |---------------------------------------------------------------------|--------------|----------|-------------|
+| [Build your serverless functions with OpenFaas on Azure](https://sajeetharan.com/2019/03/03/openfaas-revolution-is-here/) | Sajeetharan Sinnathurai | sajeetharan.com |3-Mar-2019 |
```
Comment (rgee0) [inline community.md]: OpenFaas => OpenFaaS

Comment (sajeetharan) [reply community.md]: Should be good now @rgee0

**Comment by rgee0** [comment]
Looks better, can be get a space added to the front of the date & squash the commits into 1?  Thanks.

**Comment by sajeetharan** [comment]
> Looks better, can we get a space added to the front of the date & squash the commits into 1? Thanks.

Sorted out

**Comment by alexellis** [comment]
Merged

**Pull request merged**